src/modules/iotjs_module_process.c

```diff
@@ -383,6 +383,21 @@ jerry_value_t InitProcess(void) {
 #ifdef EXPOSE_GC
   iotjs_jval_set_method(process, IOTJS_MAGIC_STRING_GC, Gc);
 #endif
+#ifdef DEBUG
+  jerry_property_descriptor_t prop_desc;
+  jerry_init_property_descriptor_fields(&prop_desc);
+
+  prop_desc.is_value_defined = true;
+  prop_desc.is_writable_defined = true;
+  prop_desc.is_configurable_defined = true;
+  prop_desc.value = jerry_create_boolean(true);
+
+  jerry_value_t property_name =
+      jerry_create_string((jerry_char_t*)IOTJS_MAGIC_STRING_DEBUG);
+  jerry_value_t prop_ret_val =
+      jerry_define_own_property(process, property_name, &prop_desc);
```

Comment:
Why don't you use `iotjs_jval_set_property_boolean` instead?

Reply:
Because the user could overwrite its value, which I don't think would be a good practice. This way the value is not writable, nor configurable.